src/topp/HighResPrecursorMassCorrector.cpp

```diff
@@ -123,7 +124,12 @@ class TOPPHiResPrecursorMassCorrector :
       registerStringOption_("nearest_peak:mz_tolerance_unit", "<choice>", "ppm", "Unit of precursor mass tolerance", false);
       setValidStrings_("nearest_peak:mz_tolerance_unit", ListUtils::create<String>("Da,ppm"));
 
-      registerOutputFile_("out_csv", "<file>", "", "Optional CSV output file for results on 'nearest_peak' algorithm (see corresponding subsection) containing columns: " + ListUtils::concatenate(ListUtils::create<String>(csv_header), ", ") + ".", false);
+      registerTOPPSubsection_("highest_intensity_peak", "Use centroided MS1 peak with the highest intensity in a certrain mass range - for precursor mass correction");
+      registerDoubleOption_("highest_intensity_peak:mz_tolerance","<num>", 0.0,"The precursor mass tolerance to find the highest intensity MS1 peak (Da) (Use 0.2 Da as default)", false);
```

Comment:
if 0.2 is the default it should be put at the position of the default value (0.0)